pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -148,6 +151,12 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 	server.PodInfraContainerImage = imageTemplate.ExpandOrDie("pod")
 	server.CPUCFSQuota = true // enable cpu cfs quota enforcement by default
 
+	if dockerClient, _, err := docker.NewHelper().GetClient(); err == nil {
```

Comment:
@deads2k If I understood correctly, I dont think so it is ignoring any config option. The main thing it is doing is setting default for docker 1.9 to parallel image pulls. It keeps the behavior as it is for docker version <=1.8 and it also honors the user set value of serializeImagepulls irrespective of docker version, and thats why the changes are before the call to cmdflags.Resolve.